### PR TITLE
cMVAv2 negative/positive configuration files

### PR DESCRIPTION
**Pull request by jpata**
Adds configuration files for negative and positive cMVAv2 b-tagger needed for scale factor measurements. Does not affect reco sequences or event content.

Replaces https://github.com/cms-sw/cmssw/pull/13007.
`scram b runtests` was successful. No more merge conflicts.

### CODE REVIEW

**Comment by cvuosalo** [comment]
@jpata: Please add to the PR description an explanation of the purpose of this PR and what differences it will create in event content and reconstructed quantities.

**Comment by jpata** [comment]
I have added some description. Please let me know if you need anything else from my side.

**Comment by cvuosalo** [comment]
@cmsbuild please test

File: RecoBTag/Combined/python/combinedMVA_cff.py [hunk 1]
```diff
@@ -11,6 +11,16 @@
 from RecoBTag.Combined.pfNegativeCombinedMVABJetTags_cfi import *
 from RecoBTag.Combined.pfPositiveCombinedMVABJetTags_cfi import *
 
-# CombinedMVA V2
+#combined MVA V2
+from RecoBTag.Combined.combinedMVAV2Computer_cfi import *
```
Comment (ferencek) [inline RecoBTag/Combined/python/combinedMVA_cff.py]: Here you should only add JetTag producers. All the computers need to go into `RecoBTag/Combined/python/combinedMVA_EventSetup_cff.py` (see #12848 for reference).

Also, don't forget to add this PR to https://twiki.cern.ch/twiki/bin/view/CMS/BTagSoftwarePullRequests Thanks

**Comment by cvuosalo** [comment]
@jpata: Please note this PR now has merge conflicts.

**Comment by slava77** [comment]
@pvmulder @jpata 
should this PR go to the backburner?
There was no feedback in this thread for almost a week now.

**Comment by imarches** [comment]
@slava77 

@jpata mentioned today (offline) that he should be able to solve the conflicts today

**Comment by jpata** [comment]
rebased to latest IB, implemented @ferencek-s comments

**Comment by jpata** [comment]
`scram b runtests` passed for me. could you please comment?

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by cvuosalo** [comment]
+1

For #13014 478e98ff2eb0da270f9df4e44884a8019582a760

Adding configuration files for negative and positive cMVAv2 b taggers.There should be no change in monitored quantities.

The code changes are satisfactory, and Jenkins [tests](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-13014/10844/summary.html) against baseline CMSSW_8_0_X_2016-01-28-2300 show no significant differences, as expected.

**Comment by jpata** [comment]
any updates needed on my side?

**Pull request merged**